app/controllers/chapters_controller.rb

```diff
@@ -49,6 +50,26 @@ def update
     render :edit, status: :unprocessable_entity
   end
 
+  def destroy
+    @chapter = Chapter.find params.require :id
+    authorize @chapter
+    @chapter.delete_chapter_and_dependents
+    return unless @chapter.save
```

Comment:
remove this since we are saving in `delete_chapter_and_dependents` so something like

`return unless @chapter.delete_chapter_and_dependents`